conf/flight_plans/tudelft/train_safe_zoo.xml

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="1.0" ground_alt="0" lat0="51.990634" lon0="4.376789" max_dist_from_home="60" name="Bebop avoid orange TU Delft Cyberzoo" security_height="0.4">
+  <header>
+    #include "subsystems/datalink/datalink.h"
```

Comment:
I think these includes are probably not needed

Reply:
I gave it a quick look and to me it looks like only the gazebo definition is not needed